decompiler/analysis/insert_lets.cpp

```diff
@@ -84,10 +84,8 @@ Form* lca_form(Form* a, Form* b, const Env& env) {
     ai--;
     bi--;
   }
-  if (!result) {
-    fmt::print("{} bad form is {}\n\n{}\n", env.func->name(), a->to_string(env), b->to_string(env));
-  }
-  ASSERT(result);
+  ASSERT_MSG(result, fmt::format("{} bad form is {}\n\n{}\n", env.func->name(), a->to_string(env),
```

Comment:
The one thing I don't like about this `ASSERT_MSG` pattern is that if you mess up the format string in `fmt::format`, it throws can exception and doesn't end up asserting (and just looks like one of the many other decompiler errors).  I am pretty tempted to modify libfmt so it just asserts in this case.  But we can worry about that later